cmd/generate.go

```diff
@@ -18,17 +20,19 @@ const (
 )
 
 type generateCmd struct {
-	apimodelPath      string
-	outputDirectory   string // can be auto-determined from clusterDefinition
-	caCertificatePath string
-	caPrivateKeyPath  string
-	classicMode       bool
-	noPrettyPrint     bool
-	parametersOnly    bool
+	translationsDirectory string
```

Comment:
Caller should not need to pass translation dir to any command ever. This decreases usability of ACS Engine. This should happen automatically. In fact do we even care of localize ACS Engine commands? Localization is primarily needed for the RP.